### PR TITLE
Fixing random test failures

### PR DESCRIPTION
**Pull request by arkpar**

### CODE REVIEW

File: ethstore/src/dir/disk.rs [hunk 1]
```diff
@@ -164,7 +164,8 @@ mod test {
 	#[test]
 	fn should_create_new_account() {
 		// given
-		let dir = env::temp_dir();
```
Comment (arkpar) [inline ethstore/src/dir/disk.rs]: This test tries to delete the temp directory root

Comment (rphmeier) [inline ethstore/src/dir/disk.rs]: why not replace its usage with `RandomTempPath`?

Comment (NikVolf) [inline ethstore/src/dir/disk.rs]: the crate dependency is already there
and is used in plenty of places

File: test.sh [hunk 2]
```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 # Running Parity Full Test Sute
 
-FEATURES="json-tests ipc"
```
Comment (NikVolf) [inline test.sh]: better disable the test
since you risk leaving main binary contain compile-time errors while all tests are passing

Comment (arkpar) [inline test.sh]: I'm not sure how many tests are affected by this

**Comment by gavofyork** [comment]
i'm sick of spurious test failures. will revert this once the grumbles are addressed. in the meantime it's useful.

**Pull request merged**